.github/workflows/CI.yml

```diff
@@ -27,11 +27,14 @@ jobs:
       run: sed -e 's/azure.archive.ubuntu.com/us.archive.ubuntu.com/g' -e t -e d /etc/apt/sources.list | sudo tee /etc/apt/sources.list.d/nonazure.list
     - name: Acquire ROS dependencies
       uses: ros-tooling/setup-ros@master
+      with:
+        # Install to avoid getting stuck on the license agreement prompt
+        install-connext: true
```

Comment:
It really shouldn't be necessary if you aren't building Connext.  I'd be interested to see what errors you got.  Nevertheless, this isn't blocking.

Reply:
I've had this problem before, but I fixed it in the action by adding `--skip-keys rti-connext-dds-5.3.1`. However, in this case, `rosdep install` won't skip `rti-connext-dds-5.3.1`, and it gets stuck on the license agreement prompt, since there's no way (that I know of) to accept the license when installing through `rosdep` (the `RTI_NC_LICENSE_ACCEPTED` env var doesn't work).

I had a hunch and tried removing the `COLCON_IGNORE` entries for `rmw_connext*`/`rosidl_typesupport_connext` (and removing the workaround), and `rosdep install` does correctly skip `rti-connext-dds-5.3.1`: https://github.com/christophebedard/rmw_cyclonedds/pull/4/checks?check_run_id=2438697165.

The `COLCON_IGNORE` trick makes `colcon` ignore the source packages here when running `colcon list --paths-only` before passing those paths to `rosdep install --from-paths`: https://github.com/ros-tooling/action-ros-ci/blob/af55a0e0a72a3343a161caa55e93b7731bf36bdc/src/action-ros-ci.ts#L195. `rosdep` then sees a dependency on connext (because of `rmw_implementation`), and installs it, ignoring the `--skip-keys`. Maybe I don't actually understand how `rosdep` works.

I'm now trying to make `colcon build` ignore a list of packages through the `colcon-defaults` input for `action-ros-ci`. I can open another PR for that though if you want.

Reply:
> I'm now trying to make `colcon build` ignore a list of packages through the `colcon-defaults` input for `action-ros-ci`.

it's not as straightforward as I thought it would be, so I'll just try again later and open a follow-up PR if it does end up working. For now this will do!